platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/buildtree/control/BuildModelParametersProvider.kt

```diff
@@ -0,0 +1,255 @@
+/*
```

Comment:
❓ 

@alllex Any changes here, or just a module/package move?

Reply:
It's a move, but the implementation of the `parameters()` method has been refactored significantly. You can also go commit-by-commit, since I put the moves between packages in separate commits compared to code changes.

Reply:
Actually, the instantiation of the model parameters changed to use the sealed class hierarchy. Anything else?